src/index.ts

```diff
@@ -95,10 +95,9 @@ const argv = yargs(process.argv.splice(2))
 	'st': {
 		alias: 's-etag',
 		group: 'Static Options:',
-		desc: "Etag generation. Set to false to disable etag generation",
+		desc: "Etag generation. Use --no-st to disable it",
```

Comment:
Add double-quotes around the mention of `--no-st` argument as it is a quote.
```suggestion
		desc: 'Etag generation. Use "--no-st" to disable it',
```